### ISSUE TITLE
data upload not working

### ISSUE THREAD

**Issue by heshamkelbashandy** [heshamkelbashandy/ess-dive-catalog#159]
In GitLab by @shreddd on Mar 22, 2018, 13:04

I see the following error when I click on "add file"
```
<?xml version="1.0" encoding="UTF-8"?><error detailCode="1100" errorCode="401" name="NotAuthorized">
    <description>The provided identity http://orcid.org/0000-0002-4775-8201 does not have permission to WRITE to the Node.</description>
</error>
```

![Screen_Shot_2018-03-22_at_1.02.22_PM](/uploads/9fe392aa90c5dd9ec61f3f86b8b61f6c/Screen_Shot_2018-03-22_at_1.02.22_PM.png)

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Mar 22, 2018, 13:48

This seems to be an issue with the syntax used in metacat.properties for `auth.allowedSubmitters`.  The line termination `\` didn't work for a list of group names.

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Mar 22, 2018, 16:56

mentioned in commit ess-dive-config@fe190eb2c3ace6dabebb7741334d58c9b0676242

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Mar 22, 2018, 17:07

assigned to @vchendrix

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Mar 22, 2018, 17:08

added ~3203216 label

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Mar 22, 2018, 17:13

Closed via ess-dive/ess-dive-config@fe190eb2

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on Mar 22, 2018, 17:13

closed
